docs/build-on-linea/tooling/node-providers.md

```diff
@@ -15,7 +15,7 @@ title: Node providers
 ## Run your own node
 
 - [Set it up yourself](../run-a-node/index.md)
-- [One-click deploy with EasyNode](https://app.easy-node.xyz/)
+- [One-click deploy with RapidNode](https://rapidnode.xyz/dashboard)
```

Comment:
Are you guys the same as EasyNode? Assuming you're different, we'd list both right?

Reply:
Yes we are in the similar business but different companies. Yes you would probably list both. Thanks!

Reply:
Cool can you make the chnage?

Reply:
My bad, I"ll do the changes